block/manager.go

```diff
@@ -53,7 +52,7 @@ const defaultBlockTime = 1 * time.Second
 const defaultLazyBlockTime = 60 * time.Second
 
 // defaultBatchRetrievalInterval is the interval at which the sequencer retrieves batches
-const defaultBatchRetrievalInterval = 1 * time.Second
+const defaultBatchRetrievalInterval = 100 * time.Microsecond
```

Comment:
when empty block production, we need to make the batch retrieval faster than the blocktime. this should be configurable in the future: https://github.com/rollkit/rollkit/issues/1832

Reply:
`Microsecond` is a little aggressive isn't it? That's going to be causing the batch retrieve loop to be cycling basically constantly. 

`Milliseconds` should be fine. If the default block time is 1s then we are still doing batches 10x faster than the block time. 

Or we should make this a factor of the block time with a maximum value. Which could eliminate the need for it to be configurable. 

Then we set it on initialization of the manager. 

something like
```go
const maxBatchRetrievalInterval = 100 time.Millisecond
const BatchRetrievalFactor = 10
func calcBatchRetrievalInterval(blockTime time.Duration) time.Duration {
   interval := blockTime / 10
   if interval > maxBatchRetrievalInterval {
      return maxBatchRetrievalInterval
   }
   return interval
}
```

Reply:
decided to use the blocktime.